contracts/AdminACLV0.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "./interfaces/0.8.x/IAdminACLV0.sol";
+import "@openzeppelin-4.7/contracts/access/Ownable.sol";
+
+/**
+ * This contract has a single superAdmin that passes all ACL checks. All checks
+ * for any other address will return false.
+ */
+contract AdminACLV0 is IAdminACLV0 {
+    string public AdminACLType = "AdminACLV0";
+
+    // superAdmin is the only address that passes any and all ACL checks
+    address public superAdmin;
+
+    constructor() {
+        superAdmin = msg.sender;
+    }
+
+    /**
+     * @dev Allows superAdmin to call transferOwnership on other contract from
+     * this contract.
+     */
+    function changeSuperAdmin(address _newSuperAdmin) external {
+        require(msg.sender == superAdmin, "Only superAdmin");
+        superAdmin = _newSuperAdmin;
+    }
+
+    /**
+     * @dev Allows superAdmin to call transferOwnership on other contract from
+     * this contract.
+     */
+    function transferOwnershipOn(address _contract, address _newOwner)
+        external
+    {
+        require(msg.sender == superAdmin, "Only superAdmin");
+        Ownable(_contract).transferOwnership(_newOwner);
+    }
+
+    /**
+     * @dev Allows superAdmin to call renounceOwnership on other contract from
+     * this contract.
+     */
+    function renounceOwnershipOn(address _contract) external {
+        require(msg.sender == superAdmin, "Only superAdmin");
+        Ownable(_contract).renounceOwnership();
+    }
```

Comment:
I'm not sure I'm understanding if/why we _need_ to expose these here at all.

Reply:
The V3 core uses OpenZeppelin's Ownable standard, which exposes these two functions gated to `onlyOwner`. Since a V3 core's owner is the AdminACL contract itself, this implementation just allows the superAdmin to instruct the AdminACL contract to call the functions on the V3 core.

An alternate implementation would be to expose functions on our V3 core that transfer/revoke ownership, and then manage the calling of those functions in our AdminACL schema. That might be a bit cleaner than this, WDYT?

Reply:
> An alternate implementation would be to expose functions on our V3 core that transfer/revoke ownership, and then manage the calling of those functions in our AdminACL schema. That might be a bit cleaner than this, WDYT?

Yeah this seems more right to me, that we'd expose this on the Core and then ACL-check the ability to do so on the AdminACL. They're functionally quite equivalent though..

Reply:
per sync discussion, prefer to keep these transfer-of-adminship functions on the AdminACL contract, but also would like to have a check that the new AdminACL contract conforms to IAdminV0, where IAdminV0 includes the `transferOwnershipOn` and `renounceOwnershipOn` functions.

@jakerockland pushed updates, tagging you for a re-review with the updates 🙌